src/stemmabench/textual_units/word.py

```diff
@@ -62,10 +65,12 @@ def mispell(self) -> str:
         Returns:
             str: The mispelled word.
         """
-        random_location = random.randrange(len(self.word))
-        return self.word[:random_location] + \
-            random.choice(LETTERS[self.language]) + \
-            self.word[random_location + 1:]
+        if len(self.word):
```

Comment:
Each time, I think
`if self.word:`
should be enough (because an empty string evaluates to false)

Reply:
- [x] use `if self.word`